layouts/index.html

```diff
@@ -23,10 +23,16 @@ <h2 class="billboard-second_text">A platform by government developers, for gover
   <a name="about"></a>
   <div class="usa-grid">
     <div class="section-content usa-content">
-      <h1>Cloud deployment is better. But that doesn’t make it easy.</h1>
+      <h1>Deploying applications that serve the public shouldn’t be slow or hard.</h1>
       <p>
-        Any federal agency deploying an app has to follow regulations, manage an ATO process, and design a robust service infrastructure. Cloud hosting saves one hassle — managing data centers — and creates another: operating cloud infrastructure and keeping it compliant. It requires a different set of skills that isn’t widely available in government yet.
+        cloud.gov gives government teams access to modern cloud deployment with fewer hassles. Now you can:
       </p>
+      <ul>
+        <li>Quickly deploy applications that comply with federal policies &mdash; without needing to manage infrastructure.</li>
+        <li>Run scalable cloud-native applications. cloud.gov provides services on top of Amazon Web Services (GovCloud), so you can use AWS services.</li>
+        <li>Try experiments: build and test prototypes without adding extra expense.</li>
```

Comment:
"without adding extra expense" feels weird. Technically it will add resource usage and managed service costs to their bill. Maybe "without obstacles" or "friction-free"?